src/components/tl/ucp/tl_ucp.h

```diff
@@ -127,10 +131,16 @@ typedef struct ucc_tl_ucp_team {
     void *                     va_base[MAX_NR_SEGMENTS];
     size_t                     base_length[MAX_NR_SEGMENTS];
     ucc_tl_ucp_worker_t *      worker;
+    ucc_tl_ucp_team_config_t   cfg;
```

Comment:
It is at team level. Each team might have different configurable options which will be set per team.

Reply:
I understand that it has to be at the team level. To clarify my previous comment, I wondered why the implementation is done inside `ucc_tl_ucp` and not once for all TLs (e.g., `topo` or `ctx_ma` could be reused by other TLs).
However, since this current PR only deals with `tl_ucp` only, I suppose that it is ok to leave it here.